src/lib.rs

```diff
@@ -261,7 +261,7 @@ For more information, check out an [example](examples/single_mem_node/main.rs#L1
 */
 
 #![cfg_attr(not(feature = "cargo-clippy"), allow(unknown_lints))]
-#![cfg_attr(feature = "cargo-clippy", feature(tool_lints))]
+#![cfg_attr(feature = "cargo-clippy")]
```

Comment:
You need to remove the whole `cfg_attr`, it doesn't make sense without the attr part